### PR TITLE
improve containers.json

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary

```SUMMARY: Content "double check and extend containers.json"```

#### Purpose of change

Attempt to help Gentleman Korg populate containers.json with relevant specifics

#### Describe the solution

* the majority of my changes were to add **modest spoil modifiers** and **moves**
* graduated cylinder had max contains 500g - changed to 1500g - I would expect a "tall, narrow glass cylinder" to be able to contain mercury (1ml : 13.5g).
* change camelbak to rigid: false
* increase max_contains_weight, usually doubling or quadrupling.
* add `"pocket_data": [ { "open_container": true, "moves": 400 } ],` to _can_food_big_unsealed_ which copy-from _can_food_big_.  
* carton_sealed and carton_unsealed have spoil 0.8, worse than bag_zipper, but slightly better than canteens with spoil 0.85 (not wed to these numbers at all, just reporting what I did)

#### Describe alternatives you've considered

- This is new, so I might misunderstand.
- ~~I'm overthinking weight_multiplies; tweaking the value to account for encumbrance/ergonomics.  So for example, liquid in a canteen is 0.98, whereas items in a plastic bag gain 5% weight.  This kind of tinkering might be unhelpful. _(I actually only did this in 2 or 3 places)_~~
- ~~I set many things to have min_item_volume: 1 or min_item_volume: 2 - in retrospect this is rather foolish, but benign~~
- a handful of times I attempted to stopwatch myself (IRL) accessing a container, but moves values are still largely arbitrary - that said, they should be consistent with one another
- the spoils values are just my best guess.  They should at least be somewhat consistent with one another.

#### Additional context

I'm sure I missed some stuff, but I'm calling this PR done.

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#containers

**Feel free to delete this, with my apologies, if that saves you time.**

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -14,13 +14,22 @@
...
         "pocket_type": "CONTAINER",
+        "min_item_volume": "1 ml",
         "watertight": true,
         "rigid": true,
         "max_contains_volume": "2500 ml",
-        "max_contains_weight": "5 kg"
+        "max_contains_weight": "10 kg",
+        "spoil_multiplier": 0.95,
+        "//magic": "tiny weight_multiplier due to ergonomics?",
+        "weight_multiplier": 0.98,
```
Comment (KorGgenT) [inline data/json/items/containers.json:28]: nothing in vanilla should have a weight_multiplier. this magically reduces the weight of the contents.

File: data/json/items/containers.json [hunk 2]
```diff
@@ -14,13 +14,22 @@
...
+        "//moves": "set a stop watch for unscrew - gulp - rescrew: 4 seconds is heroic for a 0.5 L bottle",
+        "moves": 350,
+        "//fireprot": "in the sense that liquid won't evaporate in a sealed container",
+        "fire_protection": true
```
Comment (KorGgenT) [inline data/json/items/containers.json:32]: fire protection is for protecting ammo from exploding if the container is tossed into a fire

File: data/json/items/containers.json [hunk 3]
```diff
@@ -14,13 +14,22 @@
     "material": "plastic",
     "symbol": ")",
     "color": "dark_gray",
+    "//guinea pig": "fill out all the pocket properties to wrap my head around them, hence the excessive comments ",
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
+        "min_item_volume": "1 ml",
```
Comment (KorGgenT) [inline data/json/items/containers.json:21]: this isn't really that helpful to add, and imo it just clogs up the iteminfo screen for players.

File: data/json/items/containers.json [hunk 4]
```diff
@@ -203,10 +222,13 @@
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
+        "min_item_volume": "2 ml",
```
Comment (KorGgenT) [inline data/json/items/containers.json:225]: not sure what this is for; watertight actually implies the item has no minimum item volume.

**Comment by KorGgenT** [comment]
I didn't read through all of it, but it looks like you are using fire_protection slightly incorrectly. minimum item volume shouldn't be on watertight containers, because then liquids would seep out.

**Comment by Jerimee** [comment]
@KorGgenT I committed your changes, and I wanted to flag you on this:

both can_food and can_food_unsealed have ```"open_container": true,``` - I might be taking it too literally, but how can a sealed can be an open container?  

If it is set as an open container will its sealed content fall out despite not being sealed?

I didn't change anything on this because I was confused.

**Comment by Jerimee** [comment]
getting errors that entities don't have max_contains_volume when they should have it via copy-from inheritance - unless the next errors returned are obvious, I think I'm going to leave it alone - so as not to bumble around in the dark

**Comment by KorGgenT** [comment]
> @KorGgenT I committed your changes, and I wanted to flag you on this:
> 
> both can_food and can_food_unsealed have `"open_container": true,` - I might be taking it too literally, but how can a sealed can be an open container?
> 
> If it is set as an open container will its sealed content fall out despite not being sealed?
> 
> I didn't change anything on this because I was confused.

If a container is sealed and has open_container, that means things fall out of it when it gets unsealed.

As for copy-from, you are overwriting the pocket data so you need to redefine it.

File: data/json/items/containers.json [hunk 5]
```diff
@@ -482,17 +537,18 @@
...
+    "volume": "42 L",
     "price": 0,
     "price_postapoc": 0,
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
-        "rigid": false,
+        "min_item_volume": "2 ml",
+        "rigid": true,
         "max_contains_volume": "2990 ml",
```
Comment (kevingranade) [inline data/json/items/containers.json:548]:
```suggestion
        "max_contains_volume": "42 L",
```

File: data/json/items/containers.json [hunk 6]
```diff
@@ -507,7 +563,7 @@
     "looks_like": "box_medium",
     "description": "A very large cardboard box, the sort children would have loved to hide in, when there were still children.",
     "weight": "1250 g",
-    "volume": "3150 ml",
+    "volume": "501 L",
     "//": "Volume is much lower than the actual volume of a box this size because presumably if it's in your pack, it isn't empty and full of air; and if it's in your hands, it's irrelevant what the volume is.",
```
Comment (kevingranade) [inline data/json/items/containers.json:567]:
```suggestion
```

**Comment by ZhilkinSerg** [comment]
Needs conflict resolution.

**Comment by Jerimee** [comment]
Is fine to simply close this without merge - I was trying to assist KorGgenT while not necessarily having the time and know-how to hit that mark

**Comment by Jerimee** [comment]
closing this

**Pull request closed without merging**